uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-lucene/src/main/java/org/uberfire/ext/metadata/backend/lucene/index/BaseLuceneIndex.java

```diff
@@ -94,23 +95,30 @@ public void rename( final String sourceId,
             results[ i ] = -1;
         }
 
+
+        // for each id given
```

Comment:
@porcelli Could you make sure to check the code below? I'm pretty sure I got it right, but it would be good if you could check it. 

Thanks!